todo

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+source $HOME/.todorc
+
+TODO_PATH="${TODO_PATH:=$HOME/.todo}"
+add_task(){
+
+	temp=$(cat $TODO_PATH/id.txt)
+	ID="${temp:=1}"
+	ID=$(($ID+1))
+	echo $ID >$TODO_PATH/id.txt
+	echo $ID $1 >> $TODO_PATH/todo.txt 
+	echo "$1 added sucessfuly";
+}
+
+list_task(){
+	case $1 in
+		# Handle the case when the user adds both of the option -- done
+		-a|--all)echo $(cat $TODO_PATH/todo.txt) ;;
+		-d|--done)echo $(cat $TODO_PATH/done.txt);;
+		"")echo $(cat $TODO_PATH/todo.txt)
+			echo $(cat $TODO_PATH/done.txt);;
+		*) echo bad insert ;      
+	esac
+}
+
+done_tasks(){
+	id_list=$(echo $1 | tr ',' ' ')
+	for word in $id_list
+	do
+		grep "^\<$word\>" $TODO_PATH/todo.txt >> $TODO_PATH/done.txt
+		sed "/^\<$word\>/d" $TODO_PATH/todo.txt | sponge $TODO_PATH/todo.txt					
```

Comment:
Using `sponge` is not wrong
But not using it and using the `-i` option with the `sed` command is cleaner and reduces the number of dependencies used by your script